repository/repository-web/dist/js/controllers/detailsController.js

```diff
@@ -131,9 +131,7 @@ define(["../init/appController"], function (repositoryControllers) {
                 function (result) {
                   $scope.isLoading = false;
                   $scope.message = result.data.message;
-                  if (result.data.valid) {
-                    $scope.loadDetails();
```

Comment:
I have tried figuring out whether there was any difference before removing this, but couldn't find anything relevant. 
Do you have a concrete example?
Note that model comments have little/no applicability here since saving a comment does not go through this execution path.

Reply:
As discussed, the description and last modified by fields do indeed not update when saving here, so I might as well just re-enable a full reload.